src/ext/oblivc/ot.c

```diff
@@ -1049,7 +1159,9 @@ bcipherCryptNoResize(BCipherRandomGen* gen,const char* key,int nonce,
 typedef struct
 { BCipherRandomGen *cipher;
   const char *box;
-  int n, rowBytes, *rows, k, nonce, nonceDelta, len, destParty, c;
+  HonestOTExtNonceTracker *nonce_trk;
+  size_t nonce, nonceDelta;
```

Comment:
:+1: Thanks for the type fixes.

Reply:
You're welcome. I've been trying to do 32->64 updates whenever I touch things that need them. Someone should probably take a more systematic look at things at some point in the future.